core/embed/trezorhal/stm32f4/startup_stage_2.S

```diff
@@ -6,14 +6,12 @@
   .type reset_handler, STT_FUNC
```

Comment:
@TychoVrahe We're a bit inconsistent with the `startup_xxx` file extensions (`.s` vs `.S`). I think we should unify them. Maybe `.S` is more versatile since it is processed by the C preprocessor and can contain `#ifdefs`, but we don't necessarily need that here.

Reply:
renamed to `.s`.We used to have ifdefs for model one in here, but its not needed, for now.
https://github.com/trezor/trezor-firmware/commit/6f0da0b4ccb899867f15eef0c1373be8ae38db06